Sources/muterCore/Configuration/SourceFileInfo.swift

```diff
@@ -0,0 +1,4 @@
+struct SourceFileInfo {
```

Comment:
A doc comment here would be nice, while it's fresh in our mind. Especially what the two String properties mean: a path (is it absolute?) and the file contents